config.py

```diff
@@ -532,6 +536,20 @@ def get_config(parse_args=True, cfg_path=None, options=None):
     else:
         agentConfig['ssl_certificate'] = agentConfig['ca_certs']
 
+    # Getting interval parameters for periodic metrics
```

Comment:
My bad, I thought we did because of this agentConfig.get call : https://github.com/DataDog/dd-agent/blob/master/checks/collector.py#L60 . I'll remove that at once.